src/pages/index.js

```diff
@@ -1,76 +1,56 @@
-import {nanoid} from 'nanoid';
 import Head from 'next/head';
-import {useState} from 'react';
+import {useEffect} from 'react';
 
-import Button from '../components/Button';
 import Layout from '../components/Layout';
-import useFetch from '../hooks/useFetch';
+import MovieList from '../components/MovieList';
 import useStore from '../hooks/useStore';
 
-export default function HomePage() {
-	// Data
-	const {data, loading, error} = useFetch('/api/hello');
+const API_KEY = 'cfe8f1e1a9b233b64412ec3cd0525b67';
+const GET_CONFIGURATION = 'https://api.themoviedb.org/3/configuration?api_key=' + API_KEY;
+
+//const GET_MOVIES = `https://api.themoviedb.org/3/discover/movie?api_key=${API_KEY}&language=en-US&sort_by=popularity.desc&include_adult=false&include_video=false&page=1`;
 
-	// Local state
-	const [id, setId] = useState(null);
+const GET_TV = `https://api.themoviedb.org/3/discover/tv?api_key=${API_KEY}&language=en-US&sort_by=popularity.desc&include_adult=false&include_video=false&page=1`;
 
+export default function HomePage() {
 	// Global state
-	const counter = useStore(state => state.counter);
-	const decrementCounter = useStore(state => state.decrementCounter);
-	const incrementCounter = useStore(state => state.incrementCounter);
-	const setCounter = useStore(state => state.setCounter);
+	const setConfiguration = useStore(state => state.setConfiguration);
+	const setData = useStore(state => state.setData);
+	const setConfigurationLoaded = useStore(state => state.setConfigurationLoaded);
+	const isConfigurationLoaded = useStore(state => state.isConfigurationLoaded);
+
+	useEffect(() => {
+		Promise.all([fetch(GET_TV), fetch(GET_CONFIGURATION)])
+			.then(function (responses) {
+				return Promise.all(
+					responses.map(function (response) {
+						return response.json();
+					})
+				);
+			})
+			.then(function (data) {
+				console.log('Promise.all() should be done.');
+				console.log(data);
+				setConfiguration(data[1].images);
+				setData(data[0].results);
+				setConfigurationLoaded(true);
+			})
+			.catch(function (error) {
+				console.log(error);
```

Comment:
Please do not only log the error. You should keep a record that an error occurred, so that you can display a message for the user.

Reply:
Error handling added.